bundle/API/Search/Criterion/FullText.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\SiteBundle\API\Search\Criterion;
+
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion\FullText as BaseFullText;
+use \Netgen\EzPlatformSearchExtra\API\Values\Content\Query\Criterion\FulltextSpellcheck;
```

Comment:
Extra `\` at the start.

Reply:
@emodric Fixed in https://github.com/netgen/site-bundle/pull/14/commits/f07177aaf93e125bae0700733c66fc296c8ec9c7.